docs/source/notebooks/GP-Circular.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<p>Maybe add a short doc string to explain where this formula comes from and what it does?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4082/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
it's copy paste from SO)))